src/DocumentationBrowserViewExtension/DocumentationBrowserView.xaml.cs

```diff
@@ -109,6 +113,31 @@ protected virtual void Dispose(bool disposing)
             this.documentationBrowser.DpiChanged -= DocumentationBrowser_DpiChanged;
         }
 
+        async void InitializeAsync()
+        {
+            
+            string executingPath = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+            var absoluteImagePath = Path.Combine(executingPath, FALLBACK_DOC_DIRECTORY_NAME);
+
+            //Initialize the CoreWebView2 component otherwise we can't navigate to a web page
+            await documentationBrowser.EnsureCoreWebView2Async();
+
+            //Due that the Web Browser(WebView2 - Chromium) security CORS is blocking the load of resources like images then we need to create a virtual folder in which the image are located.
+            this.documentationBrowser.CoreWebView2.SetVirtualHostNameToFolderMapping(VIRTUAL_FOLDER_MAPPING, FALLBACK_DOC_DIRECTORY_NAME, CoreWebView2HostResourceAccessKind.DenyCors);
+
+            //This will remove special characters in paths (like <img src="/path")
+            string htmlContent = HttpUtility.UrlDecode(this.viewModel.GetContent());
+           
+            //Md2Html is adding the prefix "file:///" to the image paths but due that now we are using a virtual directory is not needed
+            //Instead we will be replacing it by "http://fallback_doc/resource_name" so we will using the virtual directory
+            htmlContent = htmlContent.Replace(URI_FILE_PREFIX + absoluteImagePath, HTTP_PREFIX + VIRTUAL_FOLDER_MAPPING);
```

Comment:
Is this step better be done on the md2html tool side instead of here? I think that tool has only been used for node help docs and if we intend to use Webview2 moving forward, it's fine to modify the tool for our own consumption

Reply:
yes, please keep the logic in the tool up to date, but also make sure no one else in the repo is using it somewhere else.

Reply:
@mjkkirschner @QilongTang the Replace was removed due that now in the Md2Html tool is inserting the prefix "http://appassets" at the html image source.
I will continue checking if Md2Html tool is used for generating html files that load images
see the next commit:
https://github.com/DynamoDS/Dynamo/pull/13212/commits/b0526abe49d6bf2c9a05d56e2895e7629f84ebcc